tests/form/CountrySelect.test.tsx

```diff
@@ -3,15 +3,19 @@
 
 //tests
 import '@testing-library/jest-dom';
-import { fireEvent, render, screen } from '@testing-library/react';
-import userEvent from '@testing-library/user-event';
-import { describe, expect, it, vi } from 'vitest';
+import { render } from '@testing-library/react';
+import {
+  describe,
+  expect,
+  it,
+  vi
+} from 'vitest';
 //frui
-import CountrySelect from '../../src/form/CountrySelect.js';
+import Country from '../../src/view/Country.js';
```

Comment:
Testing wrong component. This file needs to be redone testing `CountrySelect` not `Country`